src/Observers/OperationObserver.php

```diff
@@ -34,15 +36,17 @@ public function updating(Operation $new_operation)
         if (setting('kassie.calendar.slack_integration', true) == 1 && !is_null($new_operation->integration)) {
             $old_operation = Operation::find($new_operation->id);
             if ($old_operation->is_cancelled != $new_operation->is_cancelled) {
```

Comment:
will not a `switch` be better here ?

Reply:
I don't believe so, because the conditional logic is different for each clause. I'm not sure how a switch statement would make this clearer?  For whatever it's worth, this is not new logic, I just added the check for notification settings.